### PR TITLE
test(cli): studio via mini-proxy

### PR DESCRIPTION
**Pull request by millsp**

### CODE REVIEW

File: packages/client/src/runtime/core/engines/data-proxy/utils/getClientVersion.ts [hunk 1]
```diff
@@ -32,23 +32,14 @@ async function _getClientVersion(host: string, config: EngineConfig) {
   // if it is an integration or dev version, we resolve its dataproxy
   // for this we infer the data proxy version from the engine version
   if (suffix !== undefined || clientVersion === '0.0.0' || clientVersion === 'in-memory') {
-    // we use the generated engine version to infer if we're in a test
-    if (config.engineVersion === '0000000000000000000000000000000000000000') {
+    // if the host is local, then it means we are using the mini-proxy
```
Comment (aqrln) [inline packages/client/src/runtime/core/engines/data-proxy/utils/getClientVersion.ts:35]: should we also check for `[::1]` here?

File: packages/client/src/runtime/core/engines/data-proxy/utils/getClientVersion.ts [hunk 2]
```diff
@@ -32,23 +32,14 @@ async function _getClientVersion(host: string, config: EngineConfig) {
...
+    if (host.startsWith('localhost') || host.startsWith('127.0.0.1')) {
```
Comment (aqrln) [inline packages/client/src/runtime/core/engines/data-proxy/utils/getClientVersion.ts:36]: should we also check for `[::1]` here?

File: packages/client/src/runtime/core/engines/data-proxy/utils/request.ts [hunk 3]
```diff
@@ -45,6 +45,7 @@ export async function request(
       return await customFetch(nodeFetch)(url, options)
     }
   } catch (e) {
+    console.log(url)
```
Comment (aqrln) [inline packages/client/src/runtime/core/engines/data-proxy/utils/request.ts:48]: stray `console.log`

Comment (janpio) [reply packages/client/src/runtime/core/engines/data-proxy/utils/request.ts:48]: Uh, that is a bad one @millsp

**Pull request merged**